frontend/test/playwright/e2e/external-sources.spec.ts

```diff
@@ -25,10 +25,6 @@ test("sends correct analytics events", async ({ page, context }) => {
     (event) => event.n === "SELECT_EXTERNAL_SOURCE"
   )
 
-  if (!viewEvent || !selectEvent) {
```

Comment:
Why was this removed?

Reply:
It duplicates the falsy check in the assert event payload utility.

Reply:
Got it, thank you!